crates/apps_lib/src/config/genesis/templates.rs

```diff
@@ -293,6 +293,8 @@ pub struct ChainParams<T: TemplateValidation> {
     pub implicit_vp: String,
     /// Expected number of epochs per year
     pub epochs_per_year: u64,
+    /// How many epochs it takes to transition to the next masp epoch
+    pub masp_epoch_multiplier: u64,
```

Comment:
IMO this name is a bit confusing. isn't this supposed to be more like a divider? so that masp epochs are faster than regular protocol epochs

Reply:
No we actually want masp epochs to be slower than normal ones so that we don't produce new conversions too often